src/stactools/palsar/stac.py

```diff
@@ -13,6 +16,10 @@
 
 logger = logging.getLogger(__name__)
 
+# from pystac import (Asset, CatalogType, Collection, Extent, Item, MediaType,
```

Comment:
Commented code, if not used, pls remove.

Reply:
(If this was still in draft I'd be more willing to let this slide)

Reply:
Draft for next feature. Only commented out to pass linting.